### PR TITLE
Updated Analyzer autotune id tests and other layer config tests

### PR DESCRIPTION
**Pull request by chandrams**
Updated Analyzer autotune id tests and other layer config tests

### CODE REVIEW

File: tests/scripts/da/autotune_id_tests.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ function autotune_id_tests() {
 	fi
 	
 	# Cleanup autotune
-	autotune_cleanup  | tee -a ${LOG}
+#	autotune_cleanup  | tee -a ${LOG}
```
Comment (dinogun) [inline tests/scripts/da/autotune_id_tests.sh:67]: Is this no longer needed, if so, please remove it.

File: tests/scripts/da/autotune_layer_config_id_tests.sh [hunk 2]
```diff
@@ -327,6 +302,14 @@ function validate_autotune_tunables_api() {
 			if [ "${config_id_test_name}" == "${autotune_layer_config_id_tests[3]}" ]; then
 				kubectl delete -f ${new_config_yaml} -n monitoring >> ${LOG}
 			fi
+
+			echo "Deleting autotune layer config yaml ${test_layer_config}..." | tee -a ${LOG}
+			kubectl delete -f ${config_yaml} -n monitoring | tee -a ${LOG}
```
Comment (dinogun) [inline tests/scripts/da/autotune_layer_config_id_tests.sh:307]: Ideally the `monitoring` ns needs to be set as a config variable. It should be set based on the cluster type (Eg. minikube, openshift etc)

**Review by dinogun — CHANGES_REQUESTED** [review]

File: tests/scripts/da/autotune_id_tests.sh (hunk 1 above)
Comment (chandrams) [reply tests/scripts/da/autotune_id_tests.sh]: It is needed, had commented it while testing have removed the comment.

File: tests/scripts/da/autotune_layer_config_id_tests.sh (hunk 2 above)
Comment (chandrams) [reply tests/scripts/da/autotune_layer_config_id_tests.sh]: Thanks, updated it.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**